db/migrate/20230525081252_update_created_manually_flag_on_customers.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+class UpdateCreatedManuallyFlagOnCustomers < ActiveRecord::Migration[7.0]
+  class Customer < ApplicationRecord
+    has_many :orders, class_name: "Spree::Order"
+    acts_as_taggable
+  end
+
+  module Spree
+    class Order < ApplicationRecord
+      belongs_to :customer
+      self.table_name = 'spree_orders'
+    end
+  end
+
+  def change
+    # We want to set the created_manually flag to true for all customers that
+    #  1. have at least one tag
+    #  2. or don't have any orders
+    #  3. or we created more than 1 minute before the first order
+    Customer.where(id: customers_with_at_least_one_tag)
+      .or(Customer.where.not(id: customers_with_at_least_one_order))
+      .or(
+        Customer.where(id: customers_created_more_than_1min_before_orders)
+        .where.not(id: customers_created_no_more_than_1min_before_orders)
```

Comment:
This seems redundant to me, it's the same as the previous line isn't it?

That is, they are mutually exclusive. If 

`"#{creation_time_difference_in_seconds_sql} >= 60"`

then it cannot be

`"#{creation_time_difference_in_seconds_sql} < 60"`